addons/FlashCards/src/presenter.js

```diff
@@ -362,6 +400,10 @@ function AddonFlashCards_create(){
     };
 
     presenter.nextCard = function (disregardNoLoop) {
+        if ((presenter.isFrontPlaying || presenter.isBackPlaying) && presenter.state.currentCard != originalCard) {
+            presenter.sendEndedEvent();
+        }
+        var originalCard = presenter.state.currentCard;

```

Comment:
Czy nie powinna to być zmienna globalna w presenter ? Obecnie chyba nic nie robi